packages/asset/contracts/Asset.sol

```diff
@@ -384,5 +385,19 @@ contract Asset is
         return "ASSET";
     }
 
-    uint256[49] private __gap;
+    /// @notice Returns the owner of the contract
+    /// @return address of the owner
+    function owner() external view returns (address) {
+        return _owner;
+    }
+
+    /// @notice Sets the owner of the contract
+    /// @param newOwner address of the new owner
+    function transferOwnership(address newOwner) external {
```

Comment:
why not `onlyRole(DEFAULT_ADMIN_ROLE)` ?

Reply:
I think the main reason is to be able to provide custom revert message.
Otherwise we could switch to using the modifier.